partiql-eval/src/eval/evaluable.rs

```diff
@@ -1374,6 +1411,12 @@ impl Evaluable for EvalSink {
     }
 }
 
+impl Debug for EvalSink {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        write!(f, "SINK")
+    }
+}
+
 /// Represents an evaluation operator for sub-queries, e.g. `SELECT a FROM b` in
 /// `SELECT b.c, (SELECT a FROM b) FROM books AS b`.
 #[derive(Debug)]
```

Comment:
Could add a GH issue for the other evaluator operators (e.g. `SELECT VALUE`, `GROUP BY`, `WHERE`, `HAVING`) to add a custom `Debug` for better readability